bulbs/api/views.py

```diff
@@ -466,11 +468,61 @@ def list(self, request):
                 raise Http404("No content found matching UUID")
 
             content = get_object_or_404(Content, pk=match.kwargs.get('pk'))
-            return Response(ContentSerializer().to_representation(content))
+            return Response(ContentSerializer(content).data)
         else:
             raise Http404('Must specify content "url" param')
 
 
+class SpecialCoverageResolveViewSet(viewsets.ReadOnlyModelViewSet):
+    """Retrieve special coverage containing specified content"""
+    model = SpecialCoverage
+
+    def list(self, request):
+        content_id = get_query_params(self.request).get("content_id")
+        if content_id:
+            content = get_object_or_404(Content, pk=content_id)
+            # Find special coverage via percolator
+            special_coverage_filter = {
+                "filter": {
+                    "prefix": {"_id": "specialcoverage"}
+                }
+            }
+            search_objects = type(content).search_objects
+            results = search_objects.client.percolate(
+                index=search_objects.mapping.index,
+                doc_type=search_objects.mapping.doc_type,
+                id=content.id,
+                body=special_coverage_filter,
+            )
+            # Translate perocolator results into SpecialCoverage objects
+            if results["total"]:
+                special_coverage_ids = [int(m["_id"].split(".")[-1]) for m in results["matches"]]
+                qs = SpecialCoverage.objects.filter(id__in=special_coverage_ids)
+
+                # Active Filter
```

Comment:
Best way to do these filters? I'm not filtering via percolator, thinking it's better to just do it in ORM query.